content/en/user-guide/integrations/terraform/index.md

```diff
@@ -24,7 +24,7 @@ terraform {
   required_providers {
     snowflake = {
       source  = "Snowflake-Labs/snowflake"
-      version = "~> 0.61"
+      version = "= 0.92"
```

Comment:
Having looked at [Terraform Snowflake Provider](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs) it is recommended to always use the newest version as they provide limited support. As the provider is in the experimental phase, I don't think locking the version could be a good idea, at least until a stable version with backward compatibility comes out.

Reply:
I understand your perspective, but I have a different view on this. Having a working example is important for users who want to experiment and get hands-on experience with the platform. If the current example doesn’t work which is the case now, it could discourage users from trying out the platform.

While both LocalStack Snowflake and the Snowflake Terraform provider are still in development, it is even more important to lock versions. Without version constraints, there’s a higher risk of introducing breaking changes from either side, which could cause further issues and confusion. Locking the version, for now, ensures a stable reference point for users until both tools mature and offer better backward compatibility.